src/main/java/org/spongepowered/common/mixin/core/item/MixinItemFishingRod.java

```diff
@@ -54,10 +54,12 @@ public ItemStack onItemRightClick(ItemStack itemStack, World world, EntityPlayer
             player.fishEntity.handleHookRetraction();
         } else {
             EntityFishHook fishHook = new EntityFishHook(world, player);
-            if (!Sponge.getGame().getEventManager()
+            if (world.isRemote || !Sponge.getGame().getEventManager()
```

Comment:
I just thought of another way to use per-side event handlers:
- `enum EventSide { CLIENT, SERVER, BOTH, DEFAULT }`
- If an `EventSide` is present (and not `DEFAULT`) on the `@Subscribe` annotation, that value applies.
- Otherwise, if it is present on an annotation on the class (should this be added?), that value applies.
- Otherwise, if it is present on the `@Plugin` annotation, that value applies.
- Otherwise, it defaults to `BOTH`.

@Zidane @ryantheleach How does that sound?